src/components/NavigationBar/NavigationBar.tsx

```diff
@@ -1,22 +1,84 @@
 import React, { FunctionComponent } from "react";
 import { Link } from "react-router-dom";
+import { Settings } from "react-feather";
 import logo from "./logo.svg";
 
+const urlPath = "//tradetrust.io";
+
+const navItems = [
+  {
+    id: "verify-documents",
+    label: "Verify Documents",
+    path: urlPath + "/#verify-documents",
+  },
+  {
+    id: "create-documents",
+    label: "Create Documents",
+    path: "/",
+  },
+  {
+    id: "resources",
+    label: "Resources",
+    path: urlPath + "/resources",
+  },
+  {
+    id: "faq",
+    label: "FAQ",
+    path: urlPath + "/faq",
+  },
+  {
+    id: "contact",
+    label: "Contact",
+    path: urlPath + "/#contact",
+  },
+  {
+    id: "settings",
+    label: "Settings",
+    path: urlPath + "/settings",
+  },
+];
+
 interface NavigationBar {
   logout?: () => void;
 }
 
 export const NavigationBar: FunctionComponent<NavigationBar> = ({ logout }) => {
   return (
-    <div className="container mx-auto h-16 py-2 flex items-center justify-between">
-      <Link to="/" className="h-full">
-        <img className="h-full" src={logo} alt="TradeTrust Logo" />
-      </Link>
-      {logout && (
-        <div className="cursor-pointer" onClick={logout}>
-          Logout
+    <div className="bg-navy py-6">
+      <div className="container mx-auto">
+        <div className="flex flex-wrap items-center justify-between">
+          <div className="w-full lg:w-auto">
+            <Link to="/">
+              <img className="h-12" src={logo} alt="TradeTrust Logo" />
+            </Link>
+          </div>
+          {navItems.map((item, index) => {
+            return (
+              <div
+                className={`w-auto ml-0 pr-4 pt-4 lg:pl-8 lg:pr-0 lg:py-0 ${
+                  index === 0 ? "lg:ml-auto" : ""
+                } ${index === navItems.length ? "ml-auto" : "pr-0"}`}
+                key={index}
+              >
+                {item.id === "create-documents" ? (
+                  <Link className="text-greyblue hover:text-white" to={item.path}>
+                    {item.label}
+                  </Link>
+                ) : (
+                  <a href={item.path} className="text-greyblue hover:text-white">
+                    {item.id === "settings" ? <Settings /> : item.label}
+                  </a>
+                )}
+              </div>
+            );
+          })}
+          {logout && (
+            <div className="cursor-pointer" onClick={logout}>
```

Comment:
should the logout button have the same style as the rest of the button?

Reply:
ah yes i missed that thanks